data/basics/userBalance.go

```diff
@@ -111,15 +111,33 @@ type OnlineAccountData struct {
 	VotingData
 }
 
+// GenesisAccountData contains a subset of account information that is
+// present in the genesis file.
+type GenesisAccountData struct {
```

Comment:
A little late for this suggestion, sorry about that.

I wonder if `ConsensusAccountData` would be a better name for this struct.

Reply:
No worries, changed in [12e953a](https://github.com/algorand/go-algorand/pull/5447/commits/12e953a217e46a7f9723e5cd74502d7dfeba960b)

Reply:
Yes. I think `ConsensusAccountData` is more descriptive of what data is allowed to be in the genesis allocation list, and makes more sense in the rest of the code which doesn't really care about the original genesis data anymore.